### PR TITLE
Try installing all of the octave packages available in apt.

### PR DESCRIPTION
**Pull request by moorepants**
@celine168 Please check to see if this works.

### CODE REVIEW

File: rich-default/Dockerfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 RUN apt-get update && \
-    apt install -y --no-install-recommends octave octave-control octave-image octave-io octave-optim octave-signal octave-statistics && \
+    apt install -y --no-install-recommends octave octave-* && \
```
Comment (moorepants) [inline rich-default/Dockerfile:33]: Actually, we should list them explicitly, not as a `*`.

Comment (celine168) [reply rich-default/Dockerfile:33]: There are a lot of octave packages to list them all out explicitly, if I remember correctly. The Dockerfile was able to build using `octave-*`.

**Comment by celine168** [comment]
I created a Dockerfile that incorporates this issue and https://github.com/LibreTexts/default-env/pull/24. In DockerHub, it is under the default-env with version 1.7. The main issue is just getting JupyterHub upgraded for this environment to show up.

File: rich-default/Dockerfile (hunk 1 above)
Comment (moorepants) [reply rich-default/Dockerfile:33]: Yes, it works, but it is better practice to be explicit so that the docker image is reproducible (as possible).

**Comment by celine168** [comment]
The Default Environment now runs the Dockerfile that implements this change. However, since we want to list the octave packages explicitly, we should probably make a list of octave packages to include.

**Comment by moorepants** [comment]
The list should be whatever `octave-*` grabs.

**Pull request closed without merging**